index.html

```diff
@@ -0,0 +1,176 @@
+        <div class="contents">
+          <h2>Tonic</h2>
+           <ul>
+             <li class="brand">CANOPY</li>
+             <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+             <li><img src="img/Counter.png" alt="counter"> 2015</li>
+            </ul>
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <ul class="li2">
+                 <li>html</li>
+                 <li>css</li>
+                 <li>javaScript</li>
+               </ul>
+                <button type="submit">See Project</button>
+           </div>
+        </article>
+      </aside>   
+ </div>
+  <!-- next page -->
+  <div class="continer">
+    <aside class="grid1">
+     <article>
+        <img src="img/Snapshoot(1).png" alt="Snapshoot Portfolio">
+        <div class="contents">
+          <h2>Multi-Post<br>Stories</h2>
+          <ul>
+            <li class="brand">CANOPY</li>
+            <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+            <li><img src="img/Counter.png" alt="counter"> 2015</li>
+          </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="li2">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+      <button type="submit">See Project</button>
+    </div>
+   </article>
+  </aside> 
+</div>
+<!--Next page -->
+  <div class="continer">
+    <aside class="grid1">
+      <article>
+       <img src="img/Snapshoot(2).png" alt="Snapshoot Portfolio">
+        <div class="contents">
+         <h2>Tonic</h2>
+         <ul>
+           <li class="brand">CANOPY</li>
+           <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+           <li><img src="img/Counter.png" alt="counter"> 2015</li>
+         </ul>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="li2">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+         </ul>
+         <button type="submit">See Project</button>
+       </div>
+    </article>
+   </aside>  
+ </div>
+<!--Next page -->
+  <div class="continer">
+    <aside class="grid1">
+      <article>
+      <img src="img/Snapshoot(3).png" alt="Snapshoot Portfolio">
+    <div class="contents">
+      <h2>Multi-Post<br>Stories</h2>
+      <ul>
+        <li class="brand">CANOPY</li>
+        <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+        <li><img src="img/Counter.png" alt="counter"> 2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="li2">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <button type="submit">See Project</button>
+    </div>
+    </article>
+    </aside> 
+  </div>
+<!--Next page --> 
```

Comment:
- [ ] **[REQUIRED CHANGES]**

>  Hey, according to the previous reviewer, you should use a `<section>` instead of a `<div>`. 

- Your `html` is not well structured. In order to pass the linter error you'd get if you apply the changes, you can restructure your works `<section>`. I know how hard it could be but remember the key here is for you to write clean and readable codes.  Here's a tip
```

<section class="works">
  <!-- projects-1 -->
  <ul class="">
    <li class="project-image-container">
      <img src="" alt="Tonic" class="">
    </li>

    <li>
      <h2 class=""></h2>
      <div>
        <span>CANOPY</span>
        <img src="" alt="counter">
        <span>Back End Dev</span>
        <img src="" alt="counter">
        <span>2015</span>
      </div>
      <p></p>

      <ul>
        <li>html</li>
        <li>css</li>
        <li>javaScript</li>
      </ul>
      <button>See Project</button>
    </li>
  </ul>

  <!-- projects-2 -->


  <!-- projects-3 -->


  <!-- projects-4 -->

</section>
```
- :point_up:   each `<ul>` container represent one card which in your HTML = `<div class="container"></div>`. All you need to do is to use `flex` for the layout of each card [`<div class="container">`] and use grid for your works section [`<section class="works">`] .
NOTE: _the only work here is to refactor your styling..._